sample-app/src/components/DropdownSection.tsx

```diff
@@ -3,14 +3,15 @@ import { useState, useEffect, useRef } from "react";
 
 export interface Option {
   value: string,
+  onClick: () => void,
```

Comment:
this is called on click here, but it also runs when the option is selected with the arrows keys right?